proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -117,9 +117,10 @@ message Validator {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
     (gogoproto.nullable)   = false
   ];
-
-  // True if this validator's unbonding has been stopped by an external module
+  // false iff unbonding is allowed to complete in staking EndBlock
   bool unbonding_on_hold = 12;
+  // unique id, used to distinguish unbond->rebond->unbond executions
+  uint64 unbonding_id = 13;
```

Comment:
I think the solution described [here](https://github.com/cosmos/interchain-security/issues/233#issuecomment-1198357047) would be cleaner without adding an extra field to the `Validator` struct, i.e., `validatorUnbondingCanComplete` just sets `val.UnbondingOnHold` to `false` and let the logic in `UnbondAllMatureValidators` move validators to `UNBONDED`. @jtremback what do you think?

Reply:
Actually it is not sufficient, since a validator that goes through unbond-rebond-unbond, can have its `val.UnbondingOnHold` set to `true` by the CCV module (on the second unbond) and then have this flag reset to `false` by a call to `validatorUnbondingCanComplete` for the first unbond.  

A solution could be for `val.UnbondingOnHold` to be a counter instead of a bool, i.e., 
- every call to `PutUnbondingOnHold` increments it;
- every call to `UnbondingCanComplete` decrements it;
- a validator can move to `UNBONDED` only if `val.UnbondingOnHold == 0`

Like this we allow for these functions to be used by multiple modules (in case it's needed in the future).